app/main/views/services.py

```diff
@@ -41,6 +43,10 @@ def list_services():
 
     services = Service.query.filter(
         Service.framework.has(Framework.expired == false())
+    ).order_by(
```

Comment:
I'm pretty sure the way pagination is implemented by Flask we are guaranteed not to get duplicates on the next page - I think pagination links in the API responses contain a token that references a specific query.